### ISSUE TITLE
Test for WCAG technique H70: Using frame elements to group blocks of repeated material

### ISSUE THREAD

**Issue by jessebeach** [quailjs/quail#90]
From http://www.w3.org/TR/WCAG20-TECHS/H70.html
### Description

The objective of this technique is to demonstrate how framesets can be used to group blocks of repeated material. Since most user agents and assistive technology provide a way to navigate from frame to frame, using frames to organize elements can provide a mechanism for easily bypassing blocks of repeated content. If the site uses framesets, organize the blocks of content into separate frames. Make certain that the repeated blocks of content appear in the same frame within the frameset of each Web page. In addition, each frame element must have a title attribute to describe the content of the frame. When frames are properly titled, users can use frame navigation to easily navigate between blocks of content.

This technique is appropriate when framesets are already used to organize the content of the page; other techniques are preferred for pages that are not already using framesets, because many people using assistive technology have trouble with frames . An advisory technique about using noframes is available in Success Criterion 1.1.1. 
### Procedure

If the Web page uses frames to organize content:
1. Check if repeated blocks of content are organized into separate frames.
2. Check that the frames with repeated content appear in the same location within each frameset.
### Expected Results

Checks #1 and #2 are true.

If this is a sufficient technique for a success criterion, failing this test procedure does not necessarily mean that the success criterion has not been satisfied in some other way, only that this technique has not been successfully implemented and can not be used to claim conformance.
